omnigibson/prims/joint_prim.py

```diff
@@ -835,9 +835,13 @@ def _load_state(self, state):
                 self.set_vel(state["target_vel"], target=True)
 
     def _serialize(self, state):
-        # We serialize by iterating over the keys and adding them to a list that's concatenated at the end
-        # This is a deterministic mapping because we assume the state is an dict
-        return np.concatenate(list(state.values())).astype(float)
+        return np.concatenate([
```

Comment:
was this just to make it explicit?

Reply:
Yes. It's less bug-prone. I actually need to do this for xform prim because otherwise it causes a bug.